src/app/[locale]/(pages)/accedi/errore/page.tsx

```diff
@@ -13,32 +13,77 @@ const LoginErrorPage = () => {
   const searchParams = useSearchParams();
   const t = useTranslations('ioesco');
   const errorCode = searchParams.get('errorCode');
-  const ERROR_TITLE = t('error.loginerror');
   const pushWithLocale = useLocalePush();
   const loginInfo = storageLoginInfoOps.read();
+  const DEFAULT_ERROR_TITLE = 'common.noaccesspossible';
 
-  const renderErrorSummary = (errorCode: string | null) => {
+  type errorMessage = {
+    title: string;
+    message: string;
+    hasClose: boolean;
+    hasRetry: boolean;
+  };
+
+  const renderErrorMessage = (errorCode: string | null): errorMessage => {
     if (errorCode == null) {
-      return t('error.loginerrorretry'); // Generic error
+      return {
+        title: t(DEFAULT_ERROR_TITLE),
+        message: t('error.loginerrorretry'),
+        hasClose: true,
+        hasRetry: true,
+      }; // Generic error
     }
 
     switch (errorCode) {
       case '19':
-        return t('error.credentialerror');
+        return {
+          title: t('error.toomanytries'),
+          message: t('error.credentialerror'),
+          hasClose: true,
+          hasRetry: true,
+        };
       case '20':
-        return t('error.twofactorneed');
+        return {
+          title: t(DEFAULT_ERROR_TITLE),
+          message: t('error.twofactorneed'),
```

Comment:
@Ladirico we need to update this message (see [FIGMA](https://www.figma.com/file/2BZhyc0qxN31sF6npvbeTh/IO-Esco?type=design&node-id=1479-37725&mode=design&t=lIYgSNGxpzJpLGvP-4)).